source/tutorial/routes-and-templates.md

```diff
@@ -0,0 +1,169 @@
+To demonstrate the basic setup and processing of an Ember application, this section will walk through building an Ember application for a property rental site called Super Rentals.  It will start with just a homepage, an about page and a contact page.  Let's take a look at the application from the user perspective before we get started.
+
+![super rentals homepage screenshot](../../images/routes-and-templates/ember-super-rentals-index.png)
+
+We arrive at the home page which shows a list of rentals.  From here, we will be able to navigate to an about page and a contact page.  
+
+If you haven't already, create a new Ember app called `super-rentals`.
+
+Before we start building the three pages for our app, we are going to clear out the contents of the `app/templates/application.hbs` file and only leave the `{{outlet}}` code in place.  We'll talk more about the role of the `application.hbs` file after our site has a few routes.
+
+Now, let's start with building our "about" page. Remember, when the URL path `/about` is loaded, the router will map the URL to the route handler of the same name, _about.js_.  The route handler then loads a template.  
+
+## An About Route
+
+If we run `ember help generate`, we can see a variety of tools that come with Ember for automatically generating files for various Ember resources.  We'll use the route generator to start our `about` route.
+
+```shell
+ember generate route about
+```
+
+or for short,
+
+```shell
+ember g route about
+```
+
+We can see what actions are taken by the generator:
+
+```shell
+installing route
+  create app/routes/about.js
+  create app/templates/about.hbs
+updating router
+  add route about
+installing route-test
+  create tests/unit/routes/about-test.js
+```
+
+Three new files are created: one for the route handler, one for the template the route handler will render, and a test file.  The fourth file that is touched is the router.  
+
+When we open the router, we can see that the generator has mapped a new _about_ route for us. This route will load the `about` route handler.
+
+```app/router.js
+import Ember from 'ember';
+import config from './config/environment';
+
+var Router = Ember.Router.extend({
+  location: config.locationType
+});
+
+Router.map(function() {
+  this.route('about');
+});
+
```

Comment:
We'd previously discussed not setting up change highlighting, but this seems like it could be a great place to use it.  Think it's worth doing here?
![screen shot 2015-12-01 at 9 09 49 pm](https://cloud.githubusercontent.com/assets/802505/11522022/e2586e76-986f-11e5-96d8-e01d720b4ba6.png)